src/clj/game/utils.clj

```diff
@@ -266,6 +266,11 @@
 (defn get-server-type [zone]
   (or (#{:hq :rd :archives} zone) :remote))
 
+(defn get-cid
+  "Gets the cid of a given card"
+  [card]
+  (:cid (:card card)))
```

Comment:
`(get-in card [:card :cid]` would be better?

Reply:
Good call!